### PR TITLE
Specify package versions in requirements

### PR DESCRIPTION
**Pull request by snyxan**
# Description

We should explicitly specify compatible versions of packages in requirements. This works for craftassist. Since I can't run locobot right now, could you also add versions to `locobot/requirements.txt` @soumith @anuragprat1k ?

Fixes # (issue)

## Type of change

Please check the options that are relevant.

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] Proposes a change (non-breaking change that isn't necessarily a bug)
- [ ] New feature (non-breaking change that adds a new functionality)
- [ ] Breaking change (fix or feature that would break some existing functionality downstream)
- [ ] This is a unit test
- [x] Documentation only change

## Type of requested review

- [x] I want a thorough review of the implementation.
- [ ] I want a high level review. 
- [ ] I want a deep design review.

# Testing

Tests passed locally.

# Checklist:

- [x] I have performed manual end-to-end testing of the feature in my environment.
- [ ] I have added Docstrings and comments to the code.
- [ ] I have made changes to existing documentation where needed.
- [ ] I have added tests that show that the PR is functional.
- [ ] New and existing unit tests pass locally with my changes.
- [x] I have added relevant collaborators to review the PR before merge.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -29,9 +29,9 @@ tqdm==4.41.1
 word2number==1.1
 transformers==3.3.1
 https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-2.0.0/en_core_web_sm-2.0.0.tar.gz
-coverage
-flask-cors
-flask
+coverage==5.3
+flask-cors==3.0.9
```
Comment (soumith) [inline requirements.txt:33]: exact versions seem an overkill.

I prefer if you change it to something like `flask-cors>=3.0`

**Review by soumith — CHANGES_REQUESTED** [review]
instead of exact versions, let's move to `>=` of a minimum major version.

This will allow less breakage when versions of packages upgrade underneath us

File: requirements.txt (hunk 1 above)
Comment (snyxan) [reply requirements.txt]: I was worried about package backwards compatibility issue but realized that we can rely on CI to detect it. So aggressively changed all package version req to `>=` except for `transformer` which requires a certain version.

**Review by soumith — APPROVED** [review]

**Comment by snyxan** [comment]
@kavyasrinet the conflict comes from prettytable package you added. Could you specify a minimum version for that?

**Comment by kavyasrinet** [comment]
> @kavyasrinet the conflict comes from prettytable package you added. Could you specify a minimum version for that?

Yeah just checked - `2.0.0`

**Pull request merged**